doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -191,6 +191,14 @@ Bluetooth LE
 
 .. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0
 
+DRGN-24352: hci_ipc and ipc_radio samples do not support stream flow control and drop bytes
```

Comment:
Have proper sample links.

Reply:
Probablylike this: The :zephyr:code-sample:`bluetooth_hci_ipc` sample and the :ref:`ipc_radio application